web/modules/custom/ai_screening/src/Helper/ThemeHelper.php

```diff
@@ -52,6 +54,23 @@ public function themeSuggestionsAlter(ThemeSuggestionsAlterEvent $event): void {
     if ($hook === 'select' & !empty($variables['element']['#id'])) {
       $suggestions[] = 'select__' . str_replace('-', '_', $variables['element']['#id']);
     }
+    if ($hook === 'fieldset' & !empty($variables['element']['#id'])) {
+      $suggestions[] = 'fieldset__' . str_replace(' ', '_', $variables['element']['#id']);
```

Comment:
Is it a typo that you replace space with underscore (and not dash)?

Reply:
Hmm. Wasn't used anywat so removed